go.mod

```diff
@@ -2,22 +2,30 @@ module github.com/owasp-amass/asset-db
 
 go 1.20
 
-require github.com/rubenv/sql-migrate v1.4.0
+require (
+	github.com/owasp-amass/open-asset-model v0.0.0-20230518152847-c39671e6337c
+	github.com/rubenv/sql-migrate v1.4.0
+	github.com/stretchr/testify v1.8.2
+	gorm.io/datatypes v1.2.0
+	gorm.io/driver/postgres v1.5.2
+	gorm.io/driver/sqlite v1.5.1
+	gorm.io/gorm v1.25.0
+)
 
 require (
+	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/go-gorp/gorp/v3 v3.1.0 // indirect
+	github.com/go-sql-driver/mysql v1.7.0 // indirect
 	github.com/jackc/pgpassfile v1.0.0 // indirect
 	github.com/jackc/pgservicefile v0.0.0-20221227161230-091c0ba34f0a // indirect
 	github.com/jackc/pgx/v5 v5.3.1 // indirect
 	github.com/jinzhu/inflection v1.0.0 // indirect
 	github.com/jinzhu/now v1.1.5 // indirect
 	github.com/mattn/go-sqlite3 v1.14.16 // indirect
+	github.com/pmezard/go-difflib v1.0.0 // indirect
+	github.com/stretchr/objx v0.5.0 // indirect
 	golang.org/x/crypto v0.8.0 // indirect
 	golang.org/x/text v0.9.0 // indirect
-	gorm.io/gorm v1.25.0 // indirect
-)
-
-require (
-	github.com/go-gorp/gorp/v3 v3.1.0 // indirect
-	gorm.io/driver/postgres v1.5.2
-	gorm.io/driver/sqlite v1.5.1
+	gopkg.in/yaml.v3 v3.0.1 // indirect
+	gorm.io/driver/mysql v1.4.7 // indirect
```

Comment:
weird, why is mysql an indirect dependency?

Reply:
apparently `gorm.io/driver/mysql` is a transitive dependency from `gorm.io/datatypes`

https://github.com/go-gorm/datatypes/blob/master/json.go#L14